build.gradle

```diff
@@ -102,7 +102,6 @@ task release(type: Copy, group: 'build') {
 //****************************************************************************/
 dependencies {
     compileOnly "org.elasticsearch:elasticsearch:${es_version}"
-    compileOnly group: 'com.google.errorprone', name: 'error_prone_annotations', version:'2.4.0'
```

Comment:
Why was this removed?

Reply:
Elasticsearch pulls this dependency now in this release and there was a conflict, so i removed it to use Elasticsearch's dependency